### ISSUE TITLE
Failed to start new stream error, MACOS 10.13

### ISSUE THREAD

**Issue by ExcelsiorGamingYT** [unknownskl/xbox-xcloud-client#1224]
### Describe the bug

Everytime i try to start a new stream i keep getting an error message appear saying it failed, i've tried downloading the latest version on Greenlight but its incompatible with MACOS 10.13 High Sierra will there be a fix for older versions?? As i rely on Greenlight to record my gameplay for my youtube channel??
![Screen Shot 2024-05-11 at 16 13 30](https://github.com/unknownskl/greenlight/assets/140591650/294d1070-e576-4166-a9db-88ea765914ab)

### Environment

```markdown
- OS: MACOS 10.13 High Sierra
- Device: iMac
- Greenlight version: Greenlight-2.0.0-beta15-universal
```

### New issue

- [X] I confirm that this is a new issue and not a duplicate

**Comment by placematts** [comment]
@ExcelsiorGamingYT, did you get prompted to update Greenlight to 2.0.0-beta15 and do so by clicking on the provided download button?

I think you may have run into the same issue that I did today, but I was able to overcome it by updating to the true latest release (2.3.1), which can be found here: 
`https://github.com/unknownskl/greenlight/releases`

### **Here's my story:**

I fired up Greenlight this afternoon for the first time in quite a while, so I wasn't surprised when I was prompted to update. I clicked on the provided download button and was taken to the following page for the 2.0.0-beta15 release:
`https://github.com/unknownskl/greenlight/releases/tag/v2.0.0-beta15`

Like a typical user, I didn't pay attention and thought I was getting the latest release. When I fired up 2.0.0-beta15 I was greeted by the same error as you: 
`Failed to start new stream. Error details {}
`

I did some basic troubleshooting and that's when I found your bug report. Shortly thereafter I made my way to .../greenlight/releases and found the current release is 2.3.1 and not the 2.0.0-beta15 that the Greenlight's built-in update mechanism pointed me to.

I have no idea what version of Greenlight I started with and from which I was prompted to update to 2.0.0-beta15, by the update mechanism.

### **For completeness:**

**Environment:**

```
- OS: macOS Sonoma 14.4.1
- Device: Mac Stuidio M1 Max
- Greenlight version: Greenlight-2.0.0-beta15-universal and then Greenlight-2.3.1-universal
```

**Comment by falc410** [comment]
Ran into the same Problem and the solution from @placematts works perfectly. Issue can be solved I guess.

**Comment by ExcelsiorGamingYT** [comment]
Yes tried 2.3.1 but im unable to install it as im on high sierra and i get the circle with a line through it to say its incompatible

**Comment by placematts** [comment]
> Yes tried 2.3.1 but im unable to install it as im on high sierra and i get the circle with a line through it to say its incompatible

I suggest trying some older releases (<2.3.1) and see if you can find one that is both compatible with High Sierra and free of the original issue you reported. But given how old macOS 10.13 is, you might be better off upgrading the OS, especially in the long run.

**Comment by ExcelsiorGamingYT** [comment]
@unknownskl is there going to be a fix for this??? I'm on 10.13 High Sierra and all the most recent patches/fixes are incompatible for me.
Many Thanks

**Comment by unknownskl** [comment]
Sorry, it's too complex and time consuming to support High Sierra. 
I'm running this project in my free time and maybe as you could imagine sometimes it starts to feel like an unpaid job with all the edge cases that are being brought. 
High Sierra is discontinued since Jan 2021. You are not receiving any security updates, so you should not use this OS anymore because you are at risk.

As an alternative I can suggest to install Linux and try it that way. Or maybe look at alternatives like xbplay.
